### ISSUE TITLE
The ability to set a default transform for facet anchors

### ISSUE THREAD

**Issue by epicEaston197** [Yellow-Dog-Man/Resonite-Issues#2123]
### Is your feature request related to a problem? Please describe.

The view anchor facet is too far away from me personally

### Describe the solution you'd like

I would like to be able to set a default transform for the anchor facets similar to how full body tracking remembers a "user calibration" I want a similar function so I can set "user anchors"

### Describe alternatives you've considered

Just using the avatar anchor components but this would result in me needing to add one on every single avatar as the default transform is too far away for it to be used for HUD elements

### Additional Context

Default
![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/76523692/3e17352f-445c-49fd-82ca-cfe812d3c9d2)

what I need to overwrite it to make it fit my screen
![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/76523692/6fafcdf9-28c1-47c6-9f8e-b4bd2dec3e5a)

### Requesters

@epicEaston197

**Comment by Frooxius** [comment]
This is already supported. If you don't like the default placement, you can use AvatarFacetAnchor to customize them. This includes the view ones.

From release notes:

> Added AvatarFacetAnchor (under Users/Common Avatar System)
> -- This lets you customize positioning of facet anchors on each avatar
> -- When the component isn't present, default positioning is used
> -- All full body avatars are automatically instrumented with the anchors for hands and forearms

**Comment by epicEaston197** [comment]
> This is already supported. If you don't like the default placement, you can use AvatarFacetAnchor to customize them. This includes the view ones.
> 
> From release notes:
> 
> > Added AvatarFacetAnchor (under Users/Common Avatar System)
> > -- This lets you customize positioning of facet anchors on each avatar
> > -- When the component isn't present, default positioning is used
> > -- All full body avatars are automatically instrumented with the anchors for hands and forearms

no this is not what I was referring to the "default" can't be customized

**Comment by epicEaston197** [comment]
Also if a user is in a world to where they can't load their avatar they can't move the view anchor around at all so setting a preferred default would be nice

**Comment by JackTheFoxOtter** [comment]
To make this work I think you'll need an option to force the game to always use your "default" placement. Otherwise if the avatar the world equips you already has anchors specified, how would you tell which ones you use?

**Comment by JackTheFoxOtter** [comment]
Well they get auto-injected so they will be on every avatar from my understanding. Even if they weren't you can expect most avatars to have them configured pretty soon. And a default that's only sometimes a default would be a pretty confusing UX. Also there's other issues. The forearm ones for example seem tied to the forearms of your avatar, so if you had a default offset for them, it would be in different positions for different avatars still.

**Comment by epicEaston197** [comment]
For something like a HUD interface that I'm using the view anchor for I want it to stay in place for all avatars otherwise I would have to add a view anchor with the exact same transform on every single one

**Comment by shiftyscales** [comment]
I agree with Easton that we should have some mechanism to move/modify the default placement @Frooxius.

Being able to customize placement on a per-avatar basis is nice, but doesn't supersede the need/use-case of being able to set the default placements for all avatars that do not already have their anchor placements specified.

To break it down further- there are a few underlying issues/problems I would seek to resolve in making the default placement adjustable:
- For avatars the user has not customized/set up themselves (e.g. public avatars, avatars provided by worlds that don't allow the user in their usual avatar) it allows them to have their preferred facet placements, which is especially useful if they highly customize their placements on their personal avatars
- You can disable/scale to zero any unwanted extra facet anchors (in my case, I only make use of the hand ones on the back of the hand, and view ones, and fully disable the forearm, and palm anchors)- this could be its own issue however to have a mechanism to individually toggle/disable certain anchors from use
- The default positions (at least in my headset) run off the edge of the screen for the view facets and are not fully usable- so I've had to shrink the view facets considerably
- The arm-mounted default placement have some overlap when the hands are positioned in certain positions causing the anchors to occlude each other and preventing reaching parts of the anchor

**Comment by ModernBalloonie** [comment]
Yup, I have over 30 avatars. Putting a rotation offset on all of them would be time consuming. I just basically want the z direction to be -70 degrees.
